### PR TITLE
Add Js exercise

### PR DESCRIPTION
**Pull request by maskeynihal**

### CODE REVIEW

File: leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-2.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function renderStars(n){
```
Comment (kaflesudip) [inline leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-2.js:1]: Use proper variable names intead of n, s, i.. etc.

File: leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+var fruits = [
+	{ id: 1, name: "Banana", color: "Yellow" },
+	{ id: 2, name: "Apple", color: "Red" },
+];
+
+function searchByName(fruits, keyword){
+	var foundFruit = null;
+	for (var index = 0; index < fruits.length; index++) {
+		if(fruits[index].name == keyword){
```
Comment (kaflesudip) [inline leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js:9]: `===` is preferred over `==`. Search online for why?

File: leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+var fruits = [
+	{ id: 1, name: "Banana", color: "Yellow" },
+	{ id: 2, name: "Apple", color: "Red" },
+];
+
+function searchByName(fruits, keyword){
```
Comment (kaflesudip) [inline leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js:6]: What does this return? -> boolean, strings?? If boolean has some better name like `hasFruit`.

File: leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-6.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	{
+		id: 3,
+		name: "Andrew",
+	},
+];
+
+function sortBy(array, key){
+	var copyArray = array.slice();
+	return copyArray.sort(function(a, b){
+		if(a[key] < b[key]) return -1;
```
Comment (kaflesudip) [inline leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-6.js:19]: You can reduce the lines of code with tenary operator.

File: leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js (hunk 2 above)
Comment (maskeynihal) [reply leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js:9]: I got it. Will use `===` from now.

File: leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js (hunk 3 above)
Comment (maskeynihal) [reply leapfrog/assignments/js-exercise/JS Exercise: 2-7/Exercise-4.js:6]: Thanks. will add return type and have better naming.

**Pull request merged**